ntrans_combine.py

```diff
@@ -0,0 +1,115 @@
+"""
+Reads the .csv files containing N-grams that were generated in ntrans_dataprep.py
+
+Inserts every N-gram into a collections.Counter, to determine the 10K most frequent N-grams of each sorted
+(2-grams, 3-grams, etc.).
+
+These N-grams are thereafter written to separate .csv files.
+"""
+
+import csv
+import collections
+import pathlib
+from typing import Dict, List, Tuple, Union
+
+
+NGramCounter = Tuple[str, int]
+
+
+def delete_chunkfiles():
```

Comment:
-> Union[None, Callable] ?

How do I type hint the return value of this function, sometimes just "returning", and sometimes returns by calling itself.